packages/web/src/entity/PublicDataset/index.ts

```diff
@@ -66,23 +68,25 @@ export const DatasetAnnotations = {
 
 // Limited set used for the details panel
 export const DATASET_DISPLAY_FIELDS = [
-    DatasetAnnotations.CREATION_DATE,
+    DatasetAnnotations.ORGANIZATION,
     DatasetAnnotations.RELATED_PUBLICATON,
-    DatasetAnnotations.DATASET_SIZE,
+    DatasetAnnotations.DOI,
     DatasetAnnotations.PUBLICATION_DATE,
+    DatasetAnnotations.CREATION_DATE,
+    DatasetAnnotations.DATASET_SIZE,
     DatasetAnnotations.FILE_COUNT,
-    DatasetAnnotations.DOI,
+    DatasetAnnotations.DATASET_SIZE,
```

Comment:
`DatasetAnnotations.DATASET_SIZE` is duplicated & appears twice in the dataset details panel

Reply:
Would it be reasonable to alphabetize the list?